api-server/src/server/boot/challenge.js

```diff
@@ -258,16 +258,24 @@ export function isValidChallengeCompletion(req, res, next) {
     message: 'That does not appear to be a valid challenge submission.'
   };
 
+  console.debug('console.debug');
```

Comment:
Should we hold off on deploying this logging to prod? It was meant to be temporary, but I was never able to test it bc of the deployment issue.

Maybe we can merge the branch that removes the logging directly into master?

Reply:
I'm fine if we want to merge just my branch into master.

Reply:
@testflyjets Cool--I'd be more comfortable if we did that.